test/fixtures/dialects/bigquery/dateparts.sql

```diff
@@ -14,8 +14,11 @@ SELECT
     EXTRACT(QUARTER FROM col1),
     EXTRACT(YEAR FROM col1),
     EXTRACT(ISOYEAR FROM col1),
-    WEEK(MONDAY),
```

Comment:
NOTE: Removed invalid syntax. WEEK is not a function.

```
SELECT WEEK(MONDAY)
```

causes following error

```
Function not found: WEEK at [1:8]
```